client/disputes/new-evidence/recommended-document-fields.ts

```diff
@@ -26,18 +26,83 @@ export const DOCUMENT_FIELD_KEYS = {
 	SHIPPING_DOCUMENTATION: 'shipping_documentation',
 } as const;
 
+/**
+ * Get recommended document fields for the subscription_canceled dispute reason
+ *
+ * @param {string} productType - The product type (for subscription_canceled disputes)
+ * @return {Array<{key: string, label: string}>} Array of recommended document fields
+ */
+const getRecommendedDocumentFieldsForSubscriptionCanceled = (
+	productType?: string
+): Array< RecommendedDocument > => {
+	if ( 'multiple' === productType ) {
+		return [
+			{
+				key: DOCUMENT_FIELD_KEYS.REFUND_POLICY,
+				label: __( 'Store refund policy', 'woocommerce-payments' ),
+				description: __(
+					"A screenshot of your store's refund policy.",
+					'woocommerce-payments'
+				),
+				order: 40,
+			},
+			{
+				key: DOCUMENT_FIELD_KEYS.CANCELLATION_POLICY,
+				label: __( 'Terms of service', 'woocommerce-payments' ),
+				description: __(
+					"A screenshot of your store's terms of service.",
+					'woocommerce-payments'
+				),
+				order: 50,
+			},
+		];
+	}
+
+	return [
+		{
+			key: DOCUMENT_FIELD_KEYS.ACCESS_ACTIVITY_LOG,
+			label: __( 'Subscription logs', 'woocommerce-payments' ),
+			description: __(
+				'Order notes or the history of related orders. This should clearly show successful renewals before the dispute.',
+				'woocommerce-payments'
+			),
+			order: 30,
+		},
+		{
+			key: DOCUMENT_FIELD_KEYS.REFUND_POLICY,
+			label: __( 'Store refund policy', 'woocommerce-payments' ),
+			description: __(
+				"A screenshot of your store's refund policy.",
+				'woocommerce-payments'
+			),
+			order: 40,
+		},
+		{
+			key: DOCUMENT_FIELD_KEYS.CANCELLATION_POLICY,
+			label: __( 'Terms of service', 'woocommerce-payments' ),
+			description: __(
+				"A screenshot of your store's terms of service.",
+				'woocommerce-payments'
+			),
+			order: 50,
+		},
+	];
+};
```

Comment:
Since some of the fields are the same and using the same `order`, could we reduce the duplication of those field names?

```suggestion
const getRecommendedDocumentFieldsForSubscriptionCanceled = (
	productType?: string
): Array< RecommendedDocument > => {
	const commonFields = [
		{
			key: DOCUMENT_FIELD_KEYS.REFUND_POLICY,
			label: __( 'Store refund policy', 'woocommerce-payments' ),
			description: __(
				"A screenshot of your store's refund policy.",
				'woocommerce-payments'
			),
			order: 40,
		},
		{
			key: DOCUMENT_FIELD_KEYS.CANCELLATION_POLICY,
			label: __( 'Terms of service', 'woocommerce-payments' ),
			description: __(
				"A screenshot of your store's terms of service.",
				'woocommerce-payments'
			),
			order: 50,
		},
	];

	if ( 'multiple' === productType ) {
		return commonFields;
	}

	return [
		{
			key: DOCUMENT_FIELD_KEYS.ACCESS_ACTIVITY_LOG,
			label: __( 'Subscription logs', 'woocommerce-payments' ),
			description: __(
				'Order notes or the history of related orders. This should clearly show successful renewals before the dispute.',
				'woocommerce-payments'
			),
			order: 30,
		},
		...commonFields,
	];
};
```

Reply:
I like this idea for the de-duplication and considered it too. I don't like that it obfuscates which fields will be returned in specific cases, but let me play with it a bit more.

Reply:
I had a few iterations with an agent and came up with a slightly different approach in b1eb5b89c

Reply:
I also got suggested `'multiple' !== productType` first, but changed it to be truthy like in your first case - I felt like it was easier to read :) 
Does it matter how the elements are present in the list? I used the array spread operator to ensure that the `DOCUMENT_FIELD_KEYS.ACCESS_ACTIVITY_LOG` document was first.

Reply:
Both options would work well, and I don't see any benefits of one over another with the current use case. I agree; that equation to a single case is easier to read than inversion. Changed it in 1fbd95359